private/buf/bufsync/syncer.go

```diff
@@ -629,6 +653,142 @@ func (s *syncer) readModuleAt(
 	return builtModule, nil
 }
 
+// attachOlderTags takes syncable commits for a branch already calculated, and looks back for each
+// module a given amount of commits or timestamps, syncing tags in case they were created or moved
+// after those commits were synced.
+func (s *syncer) attachOlderTags(ctx context.Context, branch string, syncableCommits []*syncableCommit) error {
```

Comment:
Nit: similarly with naming, can we call this `backfillTags`?

Reply:
Midway renaming funcs and vars, I realized the RPC is already called `AttachGitTags`, so probably that's why I chose that `attachOlderTags` naming. I'm going to rename all and let me know if it reads better even with the RPC named like that.